src/lxc/cgroups/cgfsng.c.rej

```diff
@@ -0,0 +1,53 @@
+diff a/src/lxc/cgroups/cgfsng.c b/src/lxc/cgroups/cgfsng.c	(rejected hunks)
```

Comment:
I think you don't want to commit this, right? :)

Reply:
@brauner It's my fault, thanks for https://github.com/lxc/lxc/pull/2170.